packages/c-color-mode/src/color-mode.utils.ts

```diff
@@ -74,16 +82,19 @@ export function addListener(fn: Function) {
 }
 
 export const root = {
```

Comment:
@codebender828 What exactly does the root var does here? Where is it for example called? When

TODO: update to use the setDataset() fn.